javaparser-core/src/main/java/com/github/javaparser/ast/Node.java

```diff
@@ -158,9 +203,11 @@ public final int getEndLine() {
 	 * 
 	 * @param beginColumn
 	 *            the begin column of this node
+	 * @deprecated Use setBegin()
 	 */
+	@Deprecated()
 	public final void setBeginColumn(final int beginColumn) {
-		this.beginColumn = beginColumn;
+		begin = new Position(begin == null ? 0 : begin.line, beginColumn);
```

Comment:
Done in the additional commit:
https://github.com/ptitjes/javaparser/commit/70dab5734a87c73857e0204f10f4b1311b729ede